Link the Cirrus CI configuration builder (#647)
cirruslabs/cirrus-ci-docs: docs/guide/writing-tasks.md

    **Note:** instructions within a task can only be named via a prefix (e.g. `test_script`).

> !!! tip "Task Creation for Beginners"

Reviewer: ```suggestion
!!! tip "Visual Task Creation for Beginners"
```

>     If you are just getting started and prefer a more simple way of creating tasks, the

Reviewer: "simple" here is questionable. IMO copy and pasting from examples is the simplest way.

```suggestion
    If you are just getting started and prefer a more visual way of creating tasks, there
```

Author: 'the' should not be 'there' in this case.

Reviewer: change it to work with the next line "there is a third-party"

Author: Ah

>     [Cirrus CI Configuration Builder](https://cirrusbuilder.rdil.rocks) might help.

Reviewer: ```suggestion
    is a third-party [Cirrus CI Configuration Builder](https://cirrusbuilder.rdil.rocks) for generating YAML config that might be helpful.
```
